emoji/src/main/java/com/vanniktech/emoji/Utils.java

```diff
@@ -79,6 +83,19 @@ static void backspace(@NonNull final EditText editText) {
     editText.dispatchKeyEvent(event);
   }
 
+  static List<Emoji> filterDuplicates(List<Emoji> emojis) {
+    final List<Emoji> result = new ArrayList<>(emojis);
```

Comment:
Can we reduce the number of allocations that we are doing here? I don't think we can looking at it from GitHub but I'd like to :D

Reply:
Yeah, I thought about that while doing this too, but haven't had an idea yet.

Reply:
Um, can't we just convert the list to a set?

Reply:
We can't, the duplicates are not the same equals-wise. Even if we could, I don't know if that would be a good idea performance wise, since we would have to use `LinkedHasSet` (we need ordering) and that would be quite some overhead for little gain.